lib/ClimaLandSimulations/src/utilities/pfts.jl

```diff
@@ -0,0 +1,614 @@
+    :plant_S_s,       # Specific Storativity
+    :rooting_depth    # Plant Rooting Depth
+]
+
+
+"""
+Define a PFT type that can be used to store the parameters for each PFT.
+Each PFT has a name and a list of parameters. The parameters are stored as 
+NamedTuples with the parameter name as the first element and the parameter value
+as the second element. The inner constructor checks to ensure all required
+parameters are defined for the PFT.
+"""
+struct Pft
+    name::String
+    parameters::NamedTuple
+
+    # Inner constructor to verify all required parameters are defined
+    function Pft(name, parameters)
+        defined_param_names = keys(parameters)
+        for param in param_list
+            if param ∉ defined_param_names
+                error("PFT $name is missing parameter $param")
+            end
+        end
+        new(name, parameters)
+    end
+end
+
+
+# The default PFTs used in the model use parameter values drawn from defferent 
+# sources in the literature. The sources are listed here with abbreviations that
+# are used in the comments in the code below.
+#
+#    Chen, J. M., G. Mo, J. Pisek, J. Liu, F. Deng, M. Ishizawa, 
+#    and D. Chan (2012), Effects of foliage clumping on the estimation of global
+#    terrestrial gross primary productivity, Global Biogeochem. Cycles, 26, 
+#    GB1019, doi:10.1029/2010GB003996.
+#    abb. as (Chen et al. 2012)
+#
+#    CLM5.0 Technical Note
+#    abb. as (CLM5.0)
+#
+#    Best, M. J., Pryor, M., Clark, D. B., Rooney, G. G., Essery, R. L. H., 
+#    Ménard, C. B., Edwards, J. M., Hendry, M. A., Porson, A., Gedney, N.,
+#    Mercado, L. M., Sitch, S., Blyth, E., Boucher, O., Cox, P. M., Grimmond, 
+#    C. S. B., and Harding, R. J.: The Joint UK Land Environment Simulator 
+#    (JULES), model description – Part 1: Energy and water fluxes, Geosci. Model
+#    Dev., 4, 677–699, https://doi.org/10.5194/gmd-4-677-2011, 2011.
+#    abb as (JULES Model Description) 
+#
+#    Oliver, Rebecca & Mercado, Lina & Clark, Doug & Huntingford,
+#    Chris & Taylor, Christopher & Vidale, P.L. & McGuire, Patrick & Todt, 
+#    Markus & Folwell, Sonja & Semeena, Valiyaveetil & Medlyn, Belinda. (2022).
+#    Improved representation of plant physiology in the JULES-vn5.6 land surface
+#    model: Photosynthesis, stomatal conductance and thermal acclimation. 
+#    10.5194/gmd-2022-11. 
+#    abb. as (Oliver et al. 2022)
+
+# Define the default PFTs that are used in the model.
+
+NET_Temp = Pft(
+    "Temperate Needleleaf Evergreen Tree",
+    (
+        # Radiative Transfer Model
+        Ω = 0.74,          # Chen et al. 2012 Table 1
+        α_PAR_leaf = 0.07, # CLM5.0 Table 2.3.1
+        α_NIR_leaf = 0.35, # CLM5.0 Table 2.3.1
+        τ_PAR_leaf = 0.05, # CLM5.0 Table 2.3.1
+        τ_NIR_leaf = 0.10, # CLM5.0 Table 2.3.1
+
+        # Canopy Energy Model
+        ac_canopy = 2.5e3, # Taken from site params
+
+        # Stomatal Conductance Model
+        g1 = 2.35, # CLM5.0 Table 2.9.1
```

Comment:
I need a little bit of help understanding the `g1` value units... In our code `g1` is around 150, but in the CLM tables `g1` is on the order of 1 but i don't see the unit with it: [CLM g1 table](https://escomp.github.io/ctsm-docs/versions/release-clm5.0/html/tech_note/Photosynthesis/CLM50_Tech_Note_Photosynthesis.html)

Reply:
I double checked. the typical units in the literature are sqrt(kPa). We use SI units throughout simulations which would mean we use sqrt(Pa). 1 sqrt(kPa) = 1 sqrt(kPa) * (Pa/kPa)^(1/2) sqrt(Pa) = sqrt(1000) sqrt(Pa).